cbmc_viewer/coveraget.py

```diff
@@ -142,12 +143,18 @@ def combine(self, status):
 class Coverage:
     """CBMC coverage checking results"""
 
-    def __init__(self, coverage_list):
+    def __init__(self, coverage_list, viewer_reachable=None):
         """Load CBMC coverage data."""
 
         # TODO: distinguish between coverage of source code and proof code
```

Comment:
Let's create Github issues instead of adding these comments here. https://github.com/awslabs/aws-viewer-for-cbmc/issues/37